File: cur/app/src/main/java/com/example/cur/MainActivity.java
package com.example.cur;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import android.os.Bundle;

import com.example.cur.databinding.ActivityMainBinding;

import org.graphstream.graph.Graph;
import org.graphstream.graph.implementations.DefaultGraph;
public class MainActivity extends AppCompatActivity {
    ActivityMainBinding bindingClass = bindingClass;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Graph graph = new DefaultGraph("tutorial");
        graph.addNode("A" );
        graph.addNode("B" );
        graph.addNode("C" );
        graph.addEdge("AB", "A", "B").setAttribute("length",14);
        graph.addEdge("BC", "B", "C").setAttribute("length",14);
        graph.addEdge("CA", "C", "A").setAttribute("length",14);
        graph.nodes().forEach(n -> n.setAttribute("label", n.getId()));
        graph.edges().forEach(e -> e.setAttribute("label", "" + (int) e.getNumber("length")));
    }
}

File: Learningapp/app/src/main/java/com/example/learningapp/MainActivity.kt
package com.example.learningapp

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.learningapp.databinding.ActivityMainBinding
import org.graphstream.graph.Edge
import org.graphstream.graph.Graph
import org.graphstream.graph.Node
import org.graphstream.graph.implementations.DefaultGraph

class MainActivity : AppCompatActivity() {
    lateinit var binding : ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)// содержит все элементы activity_main
        setContentView(binding.root)
    }
    fun onClickGoNewGraph(view: View){
        val intent = Intent(this,ActivityNewGraph::class.java)
        startActivity(intent)
    }

    fun onClickGoLoadGraph(view: View){
        val intent = Intent(this,LoadGraphActivity::class.java)
        startActivity(intent)
    }

}